### DISCUSSION TITLE
Docker Diff

### DISCUSSION THREAD

**Discussion by maorkuriel** [docker/extension-ideas#146] [Ideas]
Quickly see a git-like diff between one container version and another. It provides a Git-like comparison that developers are familiar with and that can be used in the following ways:

Checking Updates: Understanding what changed from one container version to another
Base image comparison: Understanding differences in base images, say from stable to latest or latest to slim versions.
Slimmed Images:  an easy way to see what was removed and kept during that process.

**Comment by ivan-curkovic** [comment]
Nice idea @maorkuriel, is there any existing tool that we could integrate as an extension?

**Reply by maorkuriel** [reply]
I came across this tool - https://github.com/GoogleContainerTools/container-diff 

But I was thinking about adding the capability to add a local docker file and compare it to a docker image / container 
WDYT?

**Comment by ivan-curkovic** [comment]
📢Annoncement: **Dockerfile Diff** 
@felipecruz91 has built this extension as a Hackathon project, and he decided to open-source it today. 
- 🔗[GitHub Repository](https://github.com/felipecruz91/dockerfile-diff-extension)
- 🖥️[Open extension in Docker Desktop](https://open.docker.com/extensions/marketplace?extensionId=felipecruz/dockerfile-diff-extension&tag=0.0.3)
 
@maorkuriel  We'd love to hear what you think about it.

**Reply by maorkuriel** [reply]
I installed the extension, I will work with it in the next couple of days and share my thoughts.
